metsim/disaggregate.py

```diff
@@ -6,90 +6,103 @@
 import pandas as pd
 
 import metsim
-from metsim.defaults import PARAMS as params
-from metsim.defaults import CONSTS as consts 
+from metsim.configuration import PARAMS as params
+from metsim.configuration import CONSTS as consts
 
-tiny_rad_fract = np.zeros(366) #This is updated during the mtclim run
 
-def disaggregate(df_daily):
+def disaggregate(df_daily, solar_geom):
     """
     TODO
     """
-    dates_hourly = pd.date_range(metsim.start, metsim.stop, freq='H') 
+    end = metsim.stop + pd.Timedelta('1 days')
+    dates_hourly = pd.date_range(metsim.start, end, freq='H')
```

Comment:
So are we hard-coding the assumption that the final output will have a time resolution that is some integer multiple of hourly?

Since we allow VIC to be run at sub-hourly time steps, we should support that in the met generation as well. Even if we do not do so in the first implementation, then we need to comment on that and open an issue for extending this to sub-hourly